client/src/ui/styles/Modal.scss

```diff
@@ -24,10 +24,45 @@
   padding: 24px;
 }
 
+.modal__fullscreen {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.modal__content--fullscreen {
+  @include forSize('tabletLandscapeUp') {
+    margin-left: 16.666%;
+    margin-right: 16.666%;
+  }
+}
+
+.modal__separator {
+  width: 100%;
+  height: 1px;
+  background-color: $borderColour;
+  margin-top: auto;
+  margin-bottom: 36px;
+}
+
 .modal__buttons {
   bottom: 0;
   display: flex;
   flex-wrap: nowrap;
   flex-direction: row;
   justify-content: space-around;
-}
\ No newline at end of file
+}
+
+.modal__buttons--fullscreen {
+  justify-content: flex-start;
```

Comment:
The button block at the bottom should be position fixed for scrollable modal content. I think this will cause some difficulties with the separator div positioned above the buttons. I'd convert this to another wrapping div instead.